contracts/modules/v2/Assessment.sol

```diff
@@ -0,0 +1,397 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.0;
+
+import "../../interfaces/INXMToken.sol";
+import "../../interfaces/ITokenController.sol";
+import "../../interfaces/IMemberRoles.sol";
+import "../../interfaces/IPool.sol";
+import "../../interfaces/IAssessment.sol";
+import "../../abstract/MasterAwareV2.sol";
+import "../../libraries/Assessment/AssessmentClaimsLib.sol";
+import "../../libraries/Assessment/AssessmentGovernanceActionsLib.sol";
+import "../../libraries/Assessment/AssessmentIncidentsLib.sol";
+import "../../libraries/Assessment/AssessmentUtilsLib.sol";
+import "../../libraries/Assessment/AssessmentVoteLib.sol";
+
+/**
+ *  Provides a way for cover owners to submit claims and redeem the payouts and facilitates
+ *  assessment processes where members decide the outcome of the events that lead to potential
+ *  payouts.
+ */
+contract Assessment is IAssessment, MasterAwareV2 {
+
+  /* ========== STATE VARIABLES ========== */
+
+  INXMToken internal nxm;
+
+  Configuration public CONFIG;
```

Comment:
lowercase `CONFIG`?

Reply:
Would the struct properties look better in lowercase as well? It could be convenient to have it as it is but I'm happy to change it if you think it's more readable otherwise.

Reply:
I'd put `config` in lowercase and the properties in uppercase. 
i.e. `config.PAYOUT_COOLDOWN_DAYS`

Reply:
For the struct properties - I'm fine with both (camelCase or UPPER_CASE). The `config` variable itself IMO looks weird uppercased.